client/definitions/colors.js

```diff
@@ -0,0 +1,4 @@
+export const COLOR_BLACK = 0x000000;
```

Comment:
definitions = constants ?
Use barell concept:
https://github.com/neu5-przenz-games/the-game/blob/staging/client/gameObjects/Tile/index.js